packages/astro-reactive-form/Form.astro

```diff
@@ -9,7 +9,6 @@ export interface Props {
 const { formGroups } = Astro.props;
 ---
 <form class="light">
-	{formGroups?.map((group) => <FieldSet group={group} />)}
```

Comment:
awesome, now the app is broken... hopefully automated tests will catch this 🤦‍♂️